src/routes/__layout.svelte

```diff
@@ -1,42 +1,7 @@
 <script lang="ts">
-  import '../app.css';
+  import { DefaultLayout } from 'src/lib/processes/layout';
```

Comment:
question: why did you introduce the `src` alias when you can import stuff from `$lib`?

Reply:
Oh, I just saw someone added this alias to the Vite config, and only after realized the useful $lib alias